package.json

```diff
@@ -71,7 +72,9 @@
       "@commitlint/config-conventional"
     ],
     "rules": {
-      "body-max-line-length": [0]
+      "body-max-line-length": [
+        0
+      ]
```

Comment:
I doubt there should be any changes to the root `package.json` for this change

Reply:
It happens everytime I run yarn so I figured why the hell not.